springboot/src/main/java/com/learning/springboot/error/TodoNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.learning.springboot.error;
+
+public class TodoNotFoundException extends RuntimeException {
+
+    public TodoNotFoundException(Long id) {
+        super("Todo id not found : " + id);
```

Comment:
Update thành "Task id " + id + " is not found."  sẽ ok hơn e

Reply:
fixed